code/names.dm

```diff
@@ -21,3 +21,8 @@ var/list/borer_names = file2list("config/names/borer.txt")
 var/list/hologram_names = file2list("config/names/hologram.txt")
 
 var/list/autoborg_silly_names = file2listExceptComments("config/names/autoborg_silly.txt")
+
+/proc/take_name(var/list/list_of_names)
```

Comment:
what is this doing here? and why does it exist when pick_n_take exists?

Reply:
Didn't realize that was a thing, sorry